go/ct/spc/specification.go

```diff
@@ -1254,6 +1254,53 @@ func getAllRules() []Rule {
 		},
 	}...)
 
+	// --- BLOBHASH ---
+
+	rules = append(rules, rulesFor(instruction{
+		op:        BLOBHASH,
+		staticGas: 3,
+		pops:      1,
+		pushes:    1,
+		conditions: []Condition{
+			RevisionBounds(R13_Cancun, NewestSupportedRevision),
+			HasBlobHash(Param(0)),
+		},
+		parameters: []Parameter{NumericParameter{}},
+		effect: func(s *st.State) {
+			indexU256 := s.Stack.Pop()
+			s.Stack.Push(NewU256FromBytes(s.TransactionContext.BlobHashes[indexU256.Uint64()][:]...))
```

Comment:
Maybe add a comment explaining why no overflow-check for the popped parameter is required. It took me a while to realize that the `HasBlobHash(..)` condition is making sure that there is no overflow.

Reply:
Well, I don't think a comment is needed since it is the condition. if this were to fail because of an overflow, a test should fail for the condition and the transaction generator before reaching this point.